controllers/userController.js

```diff
@@ -57,4 +57,64 @@ const emailCheck = async (req, res) => {
   }
 };
 
-module.exports = { signUp, signIn, emailCheck };
+const myPageShow = async (req, res) => {
+  const email = req.user.email;
+  try {
+    const result = await userService.mypage(email);
+    return res.status(200).json({ data: result });
+  } catch (err) {
+    console.log(err);
+    return res.status(err.statusCode || 500).json({ message: err.message });
+  }
+};
+
+const myAddress = async (req, res) => {
+  const email = req.user.email;
+  try {
+    const result = await userService.myAddressShow(email);
+    return res.status(200).json({ data: result });
+  } catch (err) {
+    console.log(err);
+    return res.status(err.statusCode || 500).json({ message: err.message });
+  }
+};
+
+const addressPatch = async (req, res) => {
+  try {
+    await userService.myAddressPatch(req);
+    return res.status(200).json({ message: "addressPatch Success" });
+  } catch (err) {
+    console.log(err);
+    return res.status(err.statusCode || 500).json({ message: err.message });
+  }
+};
+const myInfo = async (req, res) => {
+  const email = req.user.email;
+  try {
+    const result = await userService.myInfoShow(email);
+    return res.status(200).json({ data: result });
+  } catch (err) {
+    console.log(err);
+    return res.status(err.statusCode || 500).json({ message: err.message });
+  }
+};
+
+const pwdPatch = async (req, res) => {
+  try {
+    await userService.myPwd(req);
+    return res.status(200).json({ message: "addressPatch Success" });
```

Comment:
```suggestion
    return res.status(201).json({ message: "UPDATE_USER_PASSWORD_SUCCESS" });
```
수정에 적합한 status code를 사용해주세요. password 변경인데 reponse로 주소 수정이 성공되었다고 알려주고 있습니다.

Reply:
네 확인하고 수정했습니다 !